utils/parse/parse.go

```diff
@@ -0,0 +1,26 @@
+package parse
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+)
+
+// Port returns the port from an address
+func Port(address string) (uint, error) {
```

Comment:
This doesn't support the `user:pass@host:port` scheme, which we might want to use in the future. Let's be future-proof ;)